LiskHQ/lisk
# Merge development into feature branch - Closes #3000

### What was the problem?

Development has not been merged for a long time into the feature branch

### How did I fix it?

Merging development into the feature branch

### How to test it?

- Check that any changes made by you were correctly merged
- Build is unstable in the feature branch so if it isn't green the logs need to be checked

### Review checklist

* The PR resolves #3000
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated
